docs/contributor/adr/0007-compatibilityMode.md

```diff
@@ -0,0 +1,52 @@
+# Support for **compatibilityMode** Configuration
+
+## Status
+
+Accepted by the Istio module team
```

Comment:
Status should only be Accepted, I just saw that it's also different for the [Support for externalTrafficPolicy Configuration ADR](https://github.com/kyma-project/istio/blob/main/docs/contributor/adr/0006-support-for-externalTrafficPolicy-configuration.md), maybe you can also adapt that ADR.
```suggestion
Accepted
```